src/sdk/noobaa_s3_client/noobaa_s3_client_sdkv3.js

```diff
@@ -0,0 +1,41 @@
+/* Copyright (C) 2023 NooBaa */
+'use strict';
+
+const { S3 } = require('@aws-sdk/client-s3');
+const dbg = require('../../util/debug_module')(__filename);
+
+/**
+ * S3 Interface, S3 Class and operations in AWS SDK V3:
+ * https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-client-s3/Interface/S3/
+ * https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-client-s3/Class/S3/
+ * https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/s3/
+ */
```

Comment:
this is a leftover? if needed perhaps merge with the next comment of the class?

Reply:
I merged the comments.